english/android/marshmallow6.0.md

```diff
@@ -1,6 +1,10 @@
 You are on the right way, son!
 
-Check the updates for your smartphone now!
+> Check the updates for your smartphone now!
+
+> Its time for android 7.0 Noughat.
+
+Google has replaced it's official handset nexus into Google pixel which is currently in market with latest android 7.0(Noughat)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.